Covid-19 _Real-time_Notification/Covid.py

```diff
@@ -0,0 +1,38 @@
+from plyer import notification
+import requests
+from bs4 import BeautifulSoup
+import time
+
+def notify_user(title, message):
+    notification.notify(
+    title = title,
+    message = message,
+    app_icon = "./Covid-19_Real-time_Notification/Notify_icon.ico" ,
+    timeout = 3)
+    
+def getInfo(url):
+    r = requests.get(url)
+    return r.text
+
+
+if __name__ == '__main__':
+    while True:
+        HtmlData = getInfo('https://www.mohfw.gov.in/')
+        soup = BeautifulSoup(HtmlData, 'html.parser')
+    
+        myData=""
+        for tr in soup.find('table').find('tbody')[1].find_all('tr'):
```

Comment:
The script is not working and giving this error:

![image](https://user-images.githubusercontent.com/43691873/116179783-8d42c280-a735-11eb-9981-c68b6d38b9b9.png)